src/app/zap-templates/zcl/data-model/chip/timer-cluster.xml

```diff
@@ -0,0 +1,77 @@
+
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<configurator>
+    <domain name="CHIP" />
+
+    <bitmap name="Feature" type="bitmap32">
+        <cluster code="0x0047"/>
+        <field mask="0x01" name="RESET"/>
+    </bitmap>
+
+	<enum name="TimerStatusEnum" type="enum8">
+        <cluster code="0x0047" />
+        <item name="Running" value="0x0" />
+        <item name="Paused" value="0x1" />
+        <item name="Expired" value="0x2" />
+        <item name="Ready" value="0x3" />
+    </enum>
+
+    <cluster>
```

Comment:
Please set `<cluster apiMaturity="provisional">` until this is officially out (i.e. we can remove provisional after SVE or when we have sufficient implementations)